samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -35,6 +35,16 @@ The sample updates between the two images, either from *version 1* to *version 2
 By default, the images are saved to the `MCUboot`_ second-stage bootloader secondary slot.
 To be used by MCUboot, the downloaded images must be signed using imgtool.
 
+Configuration
+*************
+
+|config|
+
+Check and configure the following Kconfig options:
+
+- ``CONFIG_DOWNLOAD_FILE_V1`` - it configures the file name of the *version 1* of the update image.
+- ``CONFIG_DOWNLOAD_FILE_V2`` - it configures the file name of the *version 2* of the update image.
```

Comment:
These lines should go into a subsection, and I'm wondering if lines 57-61 shouldn't also be moved up?
Also, does removing SES also invalidates `   .. group-tab:: From the :file:`prj.conf` configuration file`, that is lines 68-76?